### ISSUE TITLE
Investigate production outages on 8/2/16

### ISSUE THREAD

**Issue by jseppi** [18F/calc#549]
Possibly related #519 ?

It looks like the app went down completely, giving the generic 404 error from cloud.gov.

As shown in New Relic:
<details>
<summary>`404 Not Found: Requested route ('calc.gsa.gov') does not exist.`</summary>
```
HTTP/1.1 404 Not Found
Content-Type: text/plain; charset=utf-8
Date: Tue, 02 Aug 2016 20:10:28 GMT
Server: nginx
Vary: Accept-Encoding
X-Cf-Routererror: unknown_route
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 02c963c6-6b6a-452a-65ec-db52bc43c139
Content-Length: 64
Connection: Close

404 Not Found: Requested route ('calc.gsa.gov') does not exist.
```
</details>

**Comment by toolness** [comment]
@jseppi Should we assume this was fixed by #604?

**Comment by jseppi** [comment]
Yes, closing!
